examples/isentropic_example.py

```diff
@@ -86,12 +86,9 @@
 # plotting.
 
 
-isentprs = isent_anal[0]
-isenttmp = isent_anal[1]
-isentspech = isent_anal[2]
-isentu = isent_anal[3].to('kt')
-isentv = isent_anal[4].to('kt')
-isenthgt = isent_anal[5]
+isentprs, isenttmp, isentspech, isentu, isentv, isenthgt = isent_anal[0]
```

Comment:
Looks like this is causing a test failure and was a typo in the issue. Should be:
`isentprs, isenttmp, isentspech, isentu, isentv, isenthgt = isent_anal`

Reply:
Change was made to remove [0] in isent_anal.